goss-testing/scripts/check_interface_mac_matches_bond.sh

```diff
@@ -77,6 +77,7 @@ if ! eval ip addr show dev "$INTERFACE" 1>/dev/null; then
     break
   # otherwise, it's a legit failure
   else
+    # shellcheck disable=SC2319
```

Comment:
This is a valid issue. Why are we ignoring it?

Reply:
Well, I just don't know how to get `$?` from `if ! eval ip addr show dev "$INTERFACE" 1>/dev/null; then`. Even if you put `rc=$?` right after it; it will be return code from if statement, not from `eval`. Any ideas?

Reply:
Yep, this is what was implemented in https://github.com/Cray-HPE/csm-testing/pull/456